### PR TITLE
[Portland 2023] Add CFP

### PR DESCRIPTION
**Pull request by plaindocs**
- [x] Needs blog post
- [x] Mandrill settings
- [x] Needs dates + times triple checking
- [x] CFP public on Pretalx?

----
:books: Documentation preview :books:: https://writethedocs-www--1847.org.readthedocs.build/

### CODE REVIEW

**Comment by plaindocs** [comment]
Possible fix in #1842

File: docs/conf/portland/2023/cfp.rst [hunk 1]
```diff
@@ -3,15 +3,15 @@
 Call for Proposals
 ==================
 
-Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ city }} conference, coming up on {{date.main}}. The conference is virtual, so you can speak or attend from anywhere. We will use regular {{ city }} conference times for the scheduling.
+Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next **in-person** {{ city }} conference, coming up on {{date.main}}.
```
Comment (thatdocslady) [inline docs/conf/portland/2023/cfp.rst]:
```suggestion
It's that time of year again: We’re now accepting talk proposals for our next **in-person** {{ city }} conference, coming up on {{date.main}}.
```

Comment (thatdocslady) [reply docs/conf/portland/2023/cfp.rst]: Just simplifying a bit :)

File: docs/conf/portland/2023/cfp.rst [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
 Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience. Whether you've worked on documentation for decades or you've just started this year, we want to hear from you!
 Read on to learn more about the goals of the conference and what we look for in talk proposals.
 
 {% if flagcfp %}
 In the meantime, mark your calendars:
 
-**The deadline for submitting proposals is Midnight {{tz}} on {{cfp.ends}}.**
+**The deadline for submitting proposals is 11:59 PM {{tz}} on {{cfp.ends}}.**
 
 We'll let you know if your proposal has been accepted by the end of {{cfp.notification}}.
```
Comment (thatdocslady) [inline docs/conf/portland/2023/cfp.rst:18]: - Can we make the pretalx link open in a new tab? I tried adding the suggestion directly here but because the text wasn't changed it won't let me.
- Ran through the application process, LGTM! Just a question, since this is the first in-person in a while, do we want to add another checkbox after the CoC that says something like "I confirm that I can attend the conference if accepted and will let you know ASAP if I get sick or can't travel" or similar? I know we mention it on the website, but having it right there blocking the submission button might help folks who don't read the entire CFP page or blog post...

File: docs/conf/portland/2023/news/announcing-cfp.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
+:template: {{year}}/generic.html
+
+.. post:: Dec 01, 2022
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-cfp.rst]: If we merge this sooner, maybe we can change the date so that it goes out before this date

File: docs/conf/portland/2023/news/announcing-cfp.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Announcing Call for Proposals
+=============================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`_.
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-cfp.rst]:
```suggestion
Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_.
```

File: docs/conf/portland/2023/news/announcing-cfp.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**For 2023 we are going back to an in-person event**.
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.
+
+The Call for Proposals will be open until **11:59 PM {{cfp.ends}} PST**.
```
Comment (thatdocslady) [inline docs/conf/portland/2023/news/announcing-cfp.rst:53]: Do we want to already mention the high-level health and safety information? People might (will?) ask about this before they decide whether to submit a proposal

**Review by thatdocslady — COMMENTED** [review]
Looks great, obligatory nits nitted :)

File: docs/conf/portland/2023/news/announcing-cfp.rst (hunk 5 above)
Comment (plaindocs) [reply docs/conf/portland/2023/news/announcing-cfp.rst:53]: @ericholscher wanna add some info here?

**Review by mxsasha — COMMENTED** [review]
LGTM with Mikey's comments in mind, and one other thing: some people will have issues with us requiring speakers to attend in person. In the context of our 2023 conference plans, we have accounted for that with virtual Atlantic, but that's not apparent in the CFP page or post. I feel it is a big enough issue that we should add it to both.

Rough proposal to be added to CFP page and blog post:
```
Atlantic conference in September 2023
---------------------------------------
If you want to speak at Write the Docs, but can't or don't want to attend this
in-person event, you can also submit to Write the Docs Atlantic in
September 2023. This conference is entirely virtual, and you can present or
attend from anywhere.
```

File: docs/conf/portland/2023/cfp.rst (hunk 2 above)
Comment (plaindocs) [reply docs/conf/portland/2023/cfp.rst:18]: Nice call on the confirmation. No new tab / window of the box.

Comment (plaindocs) [reply docs/conf/portland/2023/cfp.rst:18]: image

File: docs/conf/portland/2023/news/announcing-cfp.rst (hunk 3 above)
Comment (plaindocs) [reply docs/conf/portland/2023/news/announcing-cfp.rst]: :+1:

File: docs/conf/portland/2023/cfp.rst (hunk 2 above)
Comment (plaindocs) [reply docs/conf/portland/2023/cfp.rst:18]: Of course new tab possible, fixed.

**Comment by plaindocs** [comment]
@mxsasha I've added [9060d38](https://github.com/writethedocs/www/pull/1847/commits/9060d38d7c64046c7bb4cebd16d203a812d28bfa) mind taking a look?

**Comment by plaindocs** [comment]
@mxsasha would also care to sign off on dates matching Pretalx and here? :-p I've got track record in getting dates wrong.

File: docs/conf/portland/2023/cfp.rst [hunk 6]
```diff
@@ -3,15 +3,17 @@
 Call for Proposals
 ==================
 
-Hello hello, fellow documentarians! It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ city }} conference, coming up on {{date.main}}. The conference is virtual, so you can speak or attend from anywhere. We will use regular {{ city }} conference times for the scheduling.
+It's that time of year again: We’re now accepting talk proposals for our next **in-person** {{ city }} conference, coming up on {{date.main}}.
+
+(If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for `Atlantic`_ held online in a timezone as widely accessible as possible).
```
Comment (mxsasha) [inline docs/conf/portland/2023/cfp.rst]:
```suggestion
(If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for `Atlantic`_ held online between the CEST and EDT timezones).
```

File: docs/conf/portland/2023/cfp.rst [hunk 7]
```diff
@@ -183,8 +184,16 @@ Submit your proposal at {{cfp.url}}. You'll need to sign up for a Pretalx accoun
...
 
 {% endif %}
+
+.. _Atlantic:
+
+Write the Docs Atlantic, online September 2023
+----------------------------------------------
+
+If you want to speak at Write the Docs, but can't or don't want to attend an in-person event, keep an eye out for Write the Docs Atlantic, coming in September 2023.
+This conference will entirely virtual, in a timezone that works for as many people as possible, so you canm present or attend from anywhere.
```
Comment (mxsasha) [inline docs/conf/portland/2023/cfp.rst]:
```suggestion
This conference will entirely virtual, between the CEST and EDT timezones, so you can present or attend from anywhere.
```

**Review by mxsasha — COMMENTED** [review]
I like the changes, except that calling Atlantic in CEST/EDT "as widely accessible as possible" is a bit of a Euro/American centric view, so made some suggestions there.

**Comment by mxsasha** [comment]
> @mxsasha would also care to sign off on dates matching Pretalx and here? :-p I've got track record in getting dates wrong.

Dates between YAML, time in text (11:59 PM) and pretalx look consistent to me.

**Comment by plaindocs** [comment]
I reckon this is pretty much good to go!

File: docs/conf/portland/2023/news/announcing-cfp.rst [hunk 8]
```diff
@@ -0,0 +1,80 @@
+:template: {{year}}/generic.html
+
+.. post:: Dec 01, 2022
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/announcing-cfp.rst]:
```suggestion
.. post:: Nov 29, 2022
```

**Comment by plaindocs** [comment]
Switched the date to tomorrow, merging. Or would be if I had a :+1: review! @mxsasha ;-)

**Comment by plaindocs** [comment]
@mxsasha reckon this is too much? [523879a](https://github.com/writethedocs/www/pull/1847/commits/523879aaec61c9350eb58aaad8e276c19a669ba2)

**Comment by mxsasha** [comment]
> @mxsasha reckon this is too much? [523879a](https://github.com/writethedocs/www/pull/1847/commits/523879aaec61c9350eb58aaad8e276c19a669ba2)

Sounds good to me, can we also add it to the "additional speakers" field in pretalx? Because the presence of that question in the CFP makes it seem like we're inviting to multiple speakers.

**Comment by plaindocs** [comment]
Done!

**Comment by ericholscher** [comment]
Are we shipping the whole website with this launch? It doesn't look like it, which is fine if we don't want to get bogged down with it. We should definitely be linking the Health & Safety page from the CFP tho, as a minimum.

**Comment by plaindocs** [comment]
This is stuff falling between the cracks again, nobody quite sure who is doing what. I missed the Health and safety linking, will push up.

**Comment by thatdocslady** [comment]
Not sure about cracks, but I remember we said that we're going to publish the full website when we launch ticket sales. Good call on the CFP>Health page links.

**Comment by plaindocs** [comment]
I bumped the date back to the 1st December, we can go over it this evening and merge

File: docs/conf/portland/2023/health.rst [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+* that your most recent dose was at least fourteen days before the conference start
+* that your most recent dose or booster was no more than twelve months before the conference start
+
+The following are acceptable forms of proof of full vaccination:
+
+* Government-issued or government-compliant digital vaccination verification app (such as Clear or VaxYes)
+* Physical (or digital photocopy of) COVID-19 vaccination card, record, or certificate issued by a government agency, public health agency, or other authorized vaccine provider (a photo will also suffice).
+
+The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).
+
```
Comment (thatdocslady) [inline docs/conf/portland/2023/health.rst:35]:
```suggestion

If you are not sure whether your vaccination certificate is eligible, please email us before you register for the conference and we will check this for you.
```

File: docs/conf/portland/2023/health.rst [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+**Negative COVID-19 Test** **Requirement**
+
+In lieu of proof of vaccination, participants may instead show proof of a negative COVID-19 test administered by a verified provider to enter the event.
+
+* Provide a negative COVID-19 test upon entry to the event for the first time.
+* The test must be a PCR, NAAT, or Antigen test taken within 48 hours of picking up your badge on-site.
```
Comment (thatdocslady) [inline docs/conf/portland/2023/health.rst]: This might be problematic because Antigen tests should be 24hrs, and PCR test results might not be ready in 48 hours. Honestly, I'd rather have a more recent Antigen test than an old PCR test, and most governments no longer require PCR anyway so we might be able to simplify this?

File: docs/conf/portland/2023/health.rst [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+* The test may NOT be an unproctored at-home test. Tests must be administered by a verified provider, either locally in Vancouver or virtually before you travel from your home city (if time permits).
```
Comment (thatdocslady) [inline docs/conf/portland/2023/health.rst]:
```suggestion
* The test may NOT be an unproctored at-home test. Tests must be administered by a verified provider, either locally in Portland or virtually before you travel from your home city (if time permits).
```

File: docs/conf/portland/2023/health.rst [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+---------------------------------
+
+Please expect the following measures to be in place at the event:
+
+* Proof of vaccination or negative COVID-19 test required for entry
+* Event signage indicating required health and safety protocols
+* Physically spaced audience seating in certain areas of the main ballroom
+* Event participants are asked to not crowd around booths and to limit the number of attendees at a booth at any given time.
+* Wider aisles and increased event traffic flow signage to allow for physical distancing
+* Pre-packaged food and beverage wherever possible, and proper health and safety protocols for those handling food & beverage
```
Comment (thatdocslady) [inline docs/conf/portland/2023/health.rst:78]: Should we be explicit here that food = snacks and that repeat that we're not providing lunch?

File: docs/conf/portland/2023/health.rst [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+Additionally, we intend to provide free tests to attendees who need them, depending on availability. 
+
+Onsite health and safety measures
+---------------------------------
+
+Please expect the following measures to be in place at the event:
+
+* Proof of vaccination or negative COVID-19 test required for entry
+* Event signage indicating required health and safety protocols
+* Physically spaced audience seating in certain areas of the main ballroom
```
Comment (thatdocslady) [inline docs/conf/portland/2023/health.rst]: Are we still planning to have additional viewing rooms with streaming from the stage that will have open windows?

File: docs/conf/portland/2023/health.rst [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+* Event participants are asked to not crowd around booths and to limit the number of attendees at a booth at any given time.
+* Wider aisles and increased event traffic flow signage to allow for physical distancing
+* Pre-packaged food and beverage wherever possible, and proper health and safety protocols for those handling food & beverage
+* Plentiful sanitation stations onsite
+* Personal hand sanitizer bottles and wipes available for participants
+
+At a minimum, we will also be following all mandated venue and state guidelines.
+**This list, and all protocols, are subject to change as the fluid COVID-19 situation changes.**
```
Comment (thatdocslady) [inline docs/conf/portland/2023/health.rst]:
```suggestion
**This list, and all protocols, are subject to change as the fluid COVID-19 situation changes. Such changes, if they occur, will be towards the strict side, and we will not be removing requirements from this policy.**
```

**Review by thatdocslady — COMMENTED** [review]
Looks great, please see nits.

File: docs/conf/portland/2023/health.rst [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+
+If you are showing proof of vaccination to attend in person, you will need to show that you are fully vaccinated against the COVID-19 virus. We require that you:
+
+* have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
+* that your most recent dose was at least fourteen days before the conference start
+* that your most recent dose or booster was no more than twelve months before the conference start
+
+The following are acceptable forms of proof of full vaccination:
+
+* Government-issued or government-compliant digital vaccination verification app (such as Clear or VaxYes)
```
Comment (mxsasha) [inline docs/conf/portland/2023/health.rst]: If possible, it would be good to add something for people with an EU Digital Covid Certificate? It's [EU+49 countries](https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_en) so it's pretty common. As an EU person I know I have it, but have never heard of Clear or VaxYes.

File: docs/conf/portland/2023/health.rst (hunk 10 above)
Comment (plaindocs) [reply docs/conf/portland/2023/health.rst]: I'm not sure your numbers are correct.

File: docs/conf/portland/2023/health.rst (hunk 11 above)
Comment (plaindocs) [reply docs/conf/portland/2023/health.rst]: LOL what. :-D

File: docs/conf/portland/2023/health.rst [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+* Physical (or digital photocopy of) COVID-19 vaccination card, record, or certificate issued by a government agency, public health agency, or other authorized vaccine provider (a photo will also suffice).
+
+The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).
+
+**Negative COVID-19 Test** **Requirement**
+
+In lieu of proof of vaccination, participants may instead show proof of a negative COVID-19 test administered by a verified provider to enter the event.
```
Comment (mxsasha) [inline docs/conf/portland/2023/health.rst]: What is a verified provider? Is there a list?

File: docs/conf/portland/2023/health.rst (hunk 12 above)
Comment (plaindocs) [reply docs/conf/portland/2023/health.rst:78]: Not a now problem IMO.

File: docs/conf/portland/2023/health.rst (hunk 10 above)
Comment (thatdocslady) [reply docs/conf/portland/2023/health.rst]: Sorry, let me rephrase: From what I've seen, Antigen tests older than 24hrs have much higher risk of being outdated (even after 12hrs but most events and venues tend to stick to 24hrs), so it won't be as reliable. PCR test results might be delayed due to overwhelming requests at peak times in testing centers, which is why most events and venues have been using 72hrs as a max time for PCR tests.

File: docs/conf/portland/2023/health.rst (hunk 13 above)
Comment (plaindocs) [reply docs/conf/portland/2023/health.rst]: Also not a now problem IMO.

File: docs/conf/portland/2023/health.rst (hunk 14 above)
Comment (plaindocs) [reply docs/conf/portland/2023/health.rst]: I'd love to commit to this yeah.

File: docs/conf/portland/2023/health.rst (hunk 15 above)
Comment (plaindocs) [reply docs/conf/portland/2023/health.rst]: I think that's covered by "such as" but easy to add the EU one.

File: docs/conf/portland/2023/health.rst (hunk 14 above)
Comment (ericholscher) [reply docs/conf/portland/2023/health.rst]: I definitely added that, but I think some of my changes got missed 🙃. Bleh.

File: docs/conf/portland/2023/health.rst (hunk 16 above)
Comment (ericholscher) [reply docs/conf/portland/2023/health.rst]: I think basically just from a healthcase provider, but we can be more explicit here.

File: docs/conf/portland/2023/health.rst (hunk 10 above)
Comment (ericholscher) [reply docs/conf/portland/2023/health.rst]: Changed it to 72 hrs for PCR/NAAT and 24 for antigen.

File: docs/conf/portland/2023/health.rst (hunk 12 above)
Comment (ericholscher) [reply docs/conf/portland/2023/health.rst:78]: Yea, seems overly specified for now. Definitely something we can cover in other places tho.

File: docs/conf/portland/2023/health.rst (hunk 15 above)
Comment (ericholscher) [reply docs/conf/portland/2023/health.rst]: I just removed the examples? Seems easier than listing them all.

**Comment by ericholscher** [comment]
I think I hit all the feedback. I definitely wrote some of this stuff, but it must have gotten lost in the shuffle 🙃. Should be good to go, but gonna wait for folks to merge it tomorrow. I don't think we're in a huge rush, it's fine if the email goes out on Monday 👍

File: docs/conf/portland/2023/health.rst (hunk 13 above)
Comment (ericholscher) [reply docs/conf/portland/2023/health.rst]: Yea, I'm not 100% confident we can promise it, but it won't be hard to do if we have livestreaming online. Can always add it, but I don't think it's gonna make or break people's decisions up front.

**Comment by plaindocs** [comment]
Eric's updates look good. I tweaked a couple of words, bumped the date to today. :shipit:
